testing/gocryptfs/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: André Klitzing <aklitzing@gmail.com>
+# Maintainer: André Klitzing <aklitzing@gmail.com>
+pkgname=gocryptfs
+pkgver=1.3
+pkgrel=0
+pkgdesc="Encrypted overlay filesystem written in Go"
+url="https://nuetzlich.net/gocryptfs/"
+arch="all"
+license="MIT"
+makedepends="go libressl-dev fuse-dev bash"
+source="$pkgname-$pkgver.tar.gz::https://github.com/rfjakob/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/"
+
+build() {
+	cd "$builddir"
+
+	export GOPATH="$PWD/GO"
+	mkdir -p $GOPATH/src/github.com/rfjakob
+	ln -sf $PWD/$pkgname-$pkgver $GOPATH/src/github.com/rfjakob/$pkgname
+
+	go get -d github.com/rfjakob/gocryptfs
```

Comment:
Well, I added go-fuse and go-crypto... but there are a lot of stuff like this from golang.org. How to handle this?

```
../GO/src/github.com/rfjakob/gocryptfs/internal/cryptocore/cryptocore.go:12:2: cannot find package "github.com/rfjakob/eme" in any of:
        /usr/lib/go/src/github.com/rfjakob/eme (from $GOROOT)
        /home/andre/aports/testing/gocryptfs/src/GO/src/github.com/rfjakob/eme (from $GOPATH)
../GO/src/github.com/rfjakob/gocryptfs/internal/cryptocore/hkdf.go:7:2: cannot find package "golang.org/x/crypto/hkdf" in any of:
        /usr/lib/go/src/golang.org/x/crypto/hkdf (from $GOROOT)
        /home/andre/aports/testing/gocryptfs/src/GO/src/golang.org/x/crypto/hkdf (from $GOPATH)
../GO/src/github.com/rfjakob/gocryptfs/internal/configfile/scrypt.go:8:2: cannot find package "golang.org/x/crypto/scrypt" in any of:
        /usr/lib/go/src/golang.org/x/crypto/scrypt (from $GOROOT)
        /home/andre/aports/testing/gocryptfs/src/GO/src/golang.org/x/crypto/scrypt (from $GOPATH)
../GO/src/github.com/rfjakob/gocryptfs/internal/tlog/log.go:12:2: cannot find package "golang.org/x/crypto/ssh/terminal" in any of:
        /usr/lib/go/src/golang.org/x/crypto/ssh/terminal (from $GOROOT)
        /home/andre/aports/testing/gocryptfs/src/GO/src/golang.org/x/crypto/ssh/terminal (from $GOPATH)
../GO/src/github.com/rfjakob/gocryptfs/internal/fusefrontend_reverse/rfile.go:10:2: cannot find package "golang.org/x/sync/syncmap" in any of:
        /usr/lib/go/src/golang.org/x/sync/syncmap (from $GOROOT)
        /home/andre/aports/testing/gocryptfs/src/GO/src/golang.org/x/sync/syncmap (from $GOPATH)
```